.github/workflows/nightly.yml

```diff
@@ -107,6 +107,43 @@ jobs:
     - name: Run tests under valgrind
       run: ${{github.workspace}}/test/test_valgrind.sh ${{github.workspace}} ${{github.workspace}}/build ${{matrix.tool}}
 
+  Linux-FetchContent:
```

Comment:
I can you can drop Linux workflow, if you add this example to standalones execution

in `test/CMakeLists.txt`
pls add new example in line
`set(EXAMPLES ${EXAMPLES} basic custom_file_provider)`

// tested on my fork: https://github.com/lukaszstolarczuk/unified-memory-framework/actions/runs/16051712582/job/45295762065#step:11:4294

Reply:
Done